docs/ADV_CLEAN_RL.md

```diff
@@ -1,4 +1,4 @@
-# StableBaselines3
+# CleanRL
 
```

Comment:
@edbeeching further down it states: `While the default options for clean-rl work reasonably well. You may be interested in changing the hyperparameters.`. Is there a default option already for clean-rl or is this a copy paste mistake? 

Running `gdrl --trainer=clean-rl --env=gdrl --env_path=examples/AirPlatformer/bin/AirPlatformer.x86_64 --viz` does not seem to work.